docs/Homebrew-and-Python.md

```diff
@@ -26,11 +24,13 @@ Unversioned symlinks for `python`, `python-config`, `pip` etc. are installed her
 $(brew --prefix python)/libexec/bin
 ```
 
+**Warning!** The executables do not always point to the latest Python 3.y version, as there is always a delay between the newest Python 3.y release and the homebrew-core repository switching to the newest version.
```

Comment:
We allow the user to pick their own `python3` binary as we install it in `libexec/bin`. This whole section probably need rewritten to reflect that.